README.md

```diff
@@ -259,6 +259,12 @@ The NativeScript CLI respects any platform configuration files placed inside `ap
 
 Additionaly, you can modify `app/App_Resources/build.xcconfig` and `app/App_Resources/app.gradle` for adding/removing additional build properties for iOS and Android, respectively.
 
+### Modifying Entitlements File (iOS only)
+
+To specify which Capabilities are required by your App - Maps, Push Notifications, Wallet and etc. you can add or edit the `app.entitlements` file placed inside `app/App_Resources/iOS`. When building the project, the default `app/App_Resources/iOS/app.entitlements` file gets merged with all Plugins entitlement files and a new `yourAppName.entitlements` is created in the platforms directory. The path would be `app/platforms/ios/app/app.entitlements` and will be linked in the `build.xcconfig` file.
```

Comment:
`app/platforms/ios/app/app.entitlements`
Isn't it `app/platforms/ios/<application name>/<application name>.entitlements`?